doc(openshift): rework structure (#3873)
camunda/camunda-docs: versioned_docs/version-8.5/self-managed/setup/deploy/openshift/redhat-openshift.md

Deploying Camunda 8 on Red Hat OpenShift is achievable using Helm, given the appropriate configurations. However, it's important to note that the Security Context Constraints (SCCs) and Routes configurations might require slight deviations from the guidelines provided in the [general Helm deployment guide](/self-managed/setup/install.md).

## Cluster Specification

When deploying Camunda 8 on an OpenShift cluster, the cluster specification should align with your specific requirements and workload characteristics. Here's a suggested configuration to begin with:

- **Instance type:** 4 vCPUs (x86_64, >3.1 GHz), 16 GiB Memory (for example, [m5.xlarge on AWS](https://aws.amazon.com/en/ebs/general-purpose/))
- **Number of dedicated nodes:** 4
- **Volume type:** SSD volumes (with between 1000 and 3000 IOPS per volume, and a throughput of 1,000 MB/s per volume, for instance, [gp3 on AWS](https://aws.amazon.com/en/ebs/general-purpose/))

### Supported Versions

We conduct testing and ensure compatibility against the following OpenShift versions:

> - Any version not listed below hasn't been tested, and we do not guarantee compatibility.
> - The OpenShift end of support date must be in the future: we do not ensure compatibility with versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.

Reviewer: Could we move lines 26 and 27 beneath the table, so the listed versions immediately follow the testing-and-compatibility confirmation (line 24)? If we'd like to draw attention to those requirements before the table, I might change it to a statement, for example `OpenShift versions for which we ensure compatibility are provided in the following table.`

Clearer language might look something like:

```suggestion
- Compatibility is not guaranteed for any OpenShift version not explicitly listed in the table.
- Compatibility is not guaranteed for OpenShift versions no longer supported by Red Hat, as indicated in the "End of Support Date" column.
```
